go.mod

```diff
@@ -138,6 +138,7 @@ replace (
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240 // Pin client-go
 	github.com/openshift/machine-config-operator => github.com/openshift/machine-config-operator v0.0.1-0.20200130220348-e5685c0cf530 // Pin MCO so it doesn't get downgraded
 	github.com/ovirt/go-ovirt => github.com/ovirt/go-ovirt v0.0.0-20200320082526-4e97a11ff083
+	github.com/terraform-providers/terraform-provider-aws => github.com/terraform-providers/terraform-provider-aws v1.60.1-0.20200319153356-a183662640d9
```

Comment:
This is still pointing at the upstream repo.

Reply:
boo. I guess I missed this.